index.html

```diff
@@ -45,6 +203,42 @@ <h2>I'm Andrea.<br>Glad to see you!</h2>
               </li>
             </ul>
           </div>
+          <button class="efecbot">Get my resume</button>
+        </div>
+        <div class="skills">
+          <div class="langu">
+            <div class="control">
+              <p>Languajes</p>
+              <button class="more efecbot"><img src="Img/Disabled.png" alt=""></button>
+            </div>
+            <ul>
+              <li class="loglan">
+                <img src="Img/log js.png" alt="">
+                JavaScript
+              </li>
+              <li class="loglan">
+                <img src="Img/log html.png" alt="">
+                HTML
+              </li>
+              <li class="loglan">
+                <img src="Img/log css.png" alt="">
+                CSS
+              </li>
+            </ul>
+          </div>
+          <div class="frameworks">
+            <div class="control">
+              <p>Frameworks</p>
+              <button class="more efecbot"><img src="Img/enable.png" alt=""></button>
+            </div>
+          </div>
+          <div class="skills">
+            <div class="control">
+              <p>Skills</p>
```

Comment:
- [ ] Please, make sure to make the technology categories ("Languages", "Frameworks" and "Skills") inside a `<ul>` tag & each category in an `<li>` tag, as this is one of the project's requirements.
![Screenshot from 2023-04-29 05-09-33](https://user-images.githubusercontent.com/86069740/235278905-ec2a9f6a-9f1a-4138-aea7-5d00d6848798.png)

Reply:
Hello @NatiG25 
Is it something like this?
![image](https://user-images.githubusercontent.com/119966922/235281035-a2f77e81-7f2e-4627-81a6-be75adb35107.png)

Reply:
@AndreaM2429 Great job in adding the Languages category inside an <li> tag, but I recommend adding the button & the <ul> tag that contains the technology items (HTML, CSS, etc) inside the same <li> tag.

- I recommend that you do the same on other technology categories in future projects.

Reply:
Oh, like this, rigth @NatiG25 ?
![image](https://user-images.githubusercontent.com/119966922/235281688-c42b34d0-4786-40b8-a27a-87695906442a.png)

Reply:
Or you told about this part @NatiG25 
![image](https://user-images.githubusercontent.com/119966922/235281766-0caabd0a-181b-4800-9882-3df8bec28301.png)